docs/howto/krustlet-on-lke.md

```diff
@@ -0,0 +1,281 @@
+provisioned. We need access to the Kubernetes API in order to create the
+bootstrap token, so the script used to generate the bootstrap config needs to be
+run on a machine with the proper Kubernetes credentials.
+
+## Step 3: Copy bootstrap config to VM
+
+The first thing we'll need to do is copy up the assets we generated in steps 1
+and 2. Copy them to the Linode by typing:
+
+```console
+$ scp \
+  -i ${PRIVATE_KEY} \
+  ${HOME}/.krustlet/config/bootstrap.conf \
+  root@${IP}:.
+```
+
+> **NOTE** You'll need to set `PRIVATE_KEY` to the path to the private key
+corresponding to the SSH key pair that you used to create the Linode.
+
+We can then SSH into the Linode by typing:
+
+```console
+$ ssh \
+  -i ${PRIVATE_KEY} \
+  root@${IP}
+```
+
+## Step 4: Install and configure Krustlet
+
+Install the latest release of krustlet following [the install
+guide](../intro/install.md).
+
+There are two flavors of Krustlet (`krustlet-wasi` and `krustlet-wascc`), let's
+use the first:
+
+```console
+$ KUBECONFIG=${PWD}/kubeconfig krustlet-wasi \
+--hostname="krustlet" \
+--node-ip=${IP} \
+--node-name="krustlet" \
+--bootstrap-file=./bootstrap.conf \
+--cert-file=${PWD}/krustlet.crt \
+--private-key-file=${PWD}/krustlet.key
+```
+
+**NOTE** To increase the level of debugging, you may prefix the command with
+`RUST_LOG=info` or `RUST_LOG=debug`.
+
+**NOTE** The value of `${IP}` was determined in step #1.
+
+### Step 4a: Approving the serving CSR
+
+Once you have started Krustlet, there is one more manual step (though this could
+be automated depending on your setup) to perform. The client certs Krustlet
+needs are generally approved automatically by the API. However, the serving
+certs require manual approval. To do this, you'll need the hostname you
+specified for the `--hostname` flag or the output of `hostname` if you didn't
+specify anything. From another terminal that's configured to access the cluster,
+run:
+
+```console
+$ kubectl certificate approve krustlet-tls
+```
+
+> **NOTE** You will only need to do this approval step the first time Krustlet
+starts. It will generate and save all of the needed credentials to your machine
+
+You should be able to enumerate the cluster's nodes including the Krustlet by
+typing:
+
+```console
+$ kubectl get nodes
+NAME                          STATUS   ROLES    AGE   VERSION
+krustlet                      Ready    <none>   36s   0.5.0
+lke12345-12345-1234567890ab   Ready    <none>   31m   v1.18.15
+```
+
+## Step 5: Test that things work
+
+We may test that the Krustlet is working by running one of the demos:
+
+```console
+$ kubectl apply --filename=https://raw.githubusercontent.com/deislabs/krustlet/master/demos/wasi/hello-world-rust/k8s.yaml
+$ kubectl get pods
+NAME                    READY   STATUS       RESTARTS   AGE
+hello-world-wasi-rust   0/1     ExitCode:0   0          2s
+```
+
+There's an issue with the `kubectl logs` command. If you try the following, you
```

Comment:
This generally means that the API server cannot reach the network the krustlet node is on, so that should probably be called out here or the instructions should be updated to allow it to work on the same network if possible

Reply:
I think we should hold off on merging this PR until we resolve https://github.com/deislabs/krustlet/issues/492 - looks like we need to tweak some of the setup intstructions to ensure network settings are configured properly to allow `kubectl logs` to work.